tools/format.sh

```diff
@@ -57,3 +89,9 @@ if [[ ! -z "$CHANGED" ]]; then
 else
   echo "No changes."
 fi
+
+if [ "$tools_check" == "$tools_check" ]; then
```

Comment:
This doesn't look right. Also it breaks Travis.

Reply:
ups - should have been $sed_check == $tools_check :)